index.html

```diff
@@ -250,6 +250,31 @@ <h3 class="text-3">LET’S CONNECT</h3>
 
 
     </main>
+    <footer class="footer">
+        <div class="foot-image">
+            <h3 class="contactme">
+                
+                Contact me 
+            </h3>
+            <p class="contact-info">
+                If you have an application you are interested in developing,
+                a feature that you need built or a project that needs coding.
+                I’d love to help with it
+            </p>
+            <form action="https://formspree.io/f/xnqrnyla" class="contact-form" method="POST">
+                <ul class="form-item">
+                    <li ><input type="text"  placeholder="username" maxlength="30" class="username" name="username" required></li>
+                   <li><input type="email"  placeholder="Enter email" name="email"  class="email" required></li> 
+                   <li ><textarea class="message" maxlength="500" placeholder="Write your message here"></textarea></li>
+                   <li > <button class="bton get-intouch form-button" type="submit">Get It Touch</button></li>
```

Comment:
### Your Contact Form on 375px Screen Size
![contact-form-figma](https://user-images.githubusercontent.com/93039359/187660912-0a55c00e-2f91-4794-b4a9-b5829d3131c8.png)

### Figma Contact Form on 375px screen size
![contact](https://user-images.githubusercontent.com/93039359/187660948-f34cf721-8442-4e97-b168-5a4e2b834f7e.png)

- [ ] Please make your contact fields look similar to the figma template. You can see that they are going out of the screen. Please fix this issue :+1: